### DISCUSSION TITLE
How do I stick the tabbar to the top without pushing the content up?

### DISCUSSION THREAD

**Discussion by charles0122** [forus-labs/forui#211] [Q&A] [answered]
In flutter, we can achieve this effect by placing the tabbar on the appbar and the tabbarView on the body. They are associated through the Tabcontroller. In forui, Tabs does not seem to be a separable component, or I have not found a suitable way to achieve this effect.

**Comment by charles0122** [comment]
OK, I found a temporary solution, using expand and singlechildscrollview in content to wrap the view I want

**Comment by Pante** [comment, marked as answer]
Ah yup, we normally use it with an `Expanded` widget on our side too if we need it to "stick" to the top of the view. When designing `FTab`, we assumed that there isn't a large use-case for splitting the tab bar & view. That said, we're open to tweaking things if there's a strong demand.
